frontend/js/index.js

```diff
@@ -5,7 +5,7 @@ const compareButton = () => {
     checkedBoxes.forEach(checkbox => {
         link = `${link}${checkbox.value},`;
     });
-    window.location = link.substr(0,link.length-1);
+    window.location = encodeURIComponent(link.substr(0, link.length - 1));
```

Comment:
What is this supposed to do? Which part of the checkboxes is supposed to be malicious?

Reply:
I don't honestly understand the vulnerability here, but this was the recommendation on how to resolve it.

Reply:
Do you have a link?